locale/en_US/locale.po

```diff
@@ -1715,16 +1715,18 @@ msgstr "OJS Installation"
 msgid "installer.ojsUpgrade"
 msgstr "OJS Upgrade"
 
+msgid "installer.updatingInstructions"
+msgstr ""
+"\n"
+"<notification>\nIf you are upgrading an existing installation of OJS, <pkp-button element=\"a\" href=\"{$upgradeUrl}\">click here</pkp-button> to proceed.</notification>\n"
```

Comment:
Let's move the `<notification>` tag out of the translation code. That can go directly into here: https://github.com/pkp/pkp-lib/pull/6571/files#diff-68710e4a718cc0c14dbdb1cfe373f032481bd8d26d42f08d6851499e3bcd860eR18.

Also, let's change `<pkp-button>` to the `<a>` element. I think the link will be prominent enough once the whole notice is in the `<notification>` element.

Reply:
(The line breaks can also be removed.)